src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -338,8 +338,8 @@ public partial struct Boolean : System.IComparable, System.IComparable<bool>, Sy
         public override bool Equals(object obj) { return default(bool); }
         public override int GetHashCode() { return default(int); }
         public static bool Parse(string value) { return default(bool); }
-        int System.IComparable.CompareTo(object obj) { return default(int); }
-        System.TypeCode System.IConvertible.GetTypeCode() { return default(System.TypeCode); }
+        public int CompareTo(object obj) { return default(int); }
+        public System.TypeCode GetTypeCode() { return default(System.TypeCode); }
```

Comment:
I see, then what about the method `System.IConvertible.ToDateTime(System.IFormatProvider provider)` that some of these classes have? should that become implicit as well or are they explicit in the Desktop?